package.json

```diff
@@ -2,20 +2,21 @@
     "name": "pmcrypto",
     "version": "6.7.1",
     "description": "",
-    "main": "./lib/index.js",
-    "types": "./lib/index.d.ts",
-    "module": "./lib/pmcrypto.js",
+    "main": "./lib/index.ts",
     "sideEffects": false,
     "browser": {
         "buffer": false,
         "stream": false
     },
     "scripts": {
-        "test": "karma start karma.conf.js",
+        "build-worker": "webpack --config webpack.worker.js",
+        "build": "npm run build-worker",
+        "test": "npm run build-worker && karma start karma.conf.js",
         "lint": "eslint lib test --quiet",
-        "pretty": "prettier --write $(find lib -type f -name '*.js')",
+        "pretty": "prettier --write lib",
+        "preversion": "npm run build && git add -A dist",
```

Comment:
As discussed, we are not going to distribute the pre-bundled worker in the end, since dealing with the large number of generated chunks would be complicated for the webapp.